v2/tele/tele.go

```diff
@@ -86,14 +87,14 @@ func AttrKey(val string) attribute.KeyValue {
 	return attribute.String("key", val)
 }
 
-// AttrEvent creates an attribute that records the name of an event
-func AttrEvent(val string) attribute.KeyValue {
-	return attribute.String("event", val)
+// AttrInEvent creates an attribute that records the type of an event
+func AttrInEvent(t any) attribute.KeyValue {
+	return attribute.String("in_event", fmt.Sprintf("%T", t))
```

Comment:
We should implement Stringer on the events. Not in this PR but for the future

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/issues/926